web/src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import Header from "../../components/common/Header";
+import Footer from "../../components/common/Footer";
+
+class NotFoundPage extends Component {
+  render() {
+    return (
+      <div>
+        <header>
+          <Header />
```

Comment:
I saw the same on the other pages. Why the <header> is outside the Header component instead of to be inside? If both are encapsulated by a <div> anyway.

Reply:
It's because there was a page that needed a 100% width container. Now it looks a bit silly and needs to be cleaned up too!